ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/text/_cq_dialog/.content.xml

```diff
@@ -132,12 +132,15 @@
                                                     <subsuperscript
                                                         jcr:primaryType="nt:unstructured"
                                                         features="subscript,superscript"/>
+													<colorformat
+                                                        jcr:primaryType="nt:unstructured"
+                                                        features="*"/>
                                                 </rtePlugins>
                                                 <uiSettings jcr:primaryType="nt:unstructured">
                                                     <cui jcr:primaryType="nt:unstructured">
                                                         <inline
                                                             jcr:primaryType="nt:unstructured"
-                                                            toolbar="[format#bold,format#italic,format#underline,#justify,#lists,links#modifylink,links#unlink,#paraformat,findreplace#find]">
+                                                            toolbar="[format#bold,format#italic,format#underline,#justify,#lists,links#modifylink,links#unlink,#paraformat,findreplace#find,colorformat#colorPicker]">
```

Comment:
@nitheshkumar-hcl : The color picker option is not  coming for inline editing . Did see that there are some issues in getting that so we can skip for now.

Reply:
Removed the color picker option for inline editing as per the review comments